pkg/cmd/pulumi/operations/io.go

```diff
@@ -145,3 +147,16 @@ func configureCopilotOptions(copilotEnabledFlag bool, cmd *cobra.Command, displa
 	displayOpts.CopilotSummaryModel = env.CopilotSummaryModel.Value()
 	displayOpts.CopilotSummaryMaxLen = env.CopilotSummaryMaxLen.Value()
 }
+
+func handleRefreshFlag(variable declared.Value) string {
```

Comment:
We could even make this more generic as `handleBoolFlag`, since it's not really specific to refresh.